doc/source/whatsnew/v0.25.0.rst

```diff
@@ -126,6 +126,7 @@ Other Enhancements
 - :class:`RangeIndex` has gained :attr:`~RangeIndex.start`, :attr:`~RangeIndex.stop`, and :attr:`~RangeIndex.step` attributes (:issue:`25710`)
 - :class:`datetime.timezone` objects are now supported as arguments to timezone methods and constructors (:issue:`25065`)
 - :meth:`DataFrame.query` and :meth:`DataFrame.eval` now supports quoting column names with backticks to refer to names with spaces (:issue:`6508`)
+- :meth:`DataFrame.to_latex` now accepts ``caption`` and ``label`` arguments (:issue:`25436`)
```

Comment:
@jeschwar apologies, can you move this to `0.26.0.rst`?

Reply:
@TomAugspurger sure no problem.  It looks like I will have to create the file `pandas/doc/source/whatsnew/0.26.0.rst`; is that correct?

Reply:
Sorry @jeschwar, in the meantime we decided to just use `1.0.0.rst`. Can you update that one more time?